SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3225,6 +3238,8 @@ void USpatialNetDriver::TryFinishStartup()
 				bIsReadyToStart = true;
 				Connection->SetStartupComplete();
 
+				CreateAndInitializeCoreClassesAfterStartup();
```

Comment:
The RPCService won't exist now prior to BeginPlay, I wonder what will happen if you attempt to call a multicast/client rpc during InitializeComponents or similar. Could you test it?

Reply:
Checked this - everything proceeds correctly; as a sidenote, the behavior is very similar between Native and Spatial.